agents.py

```diff
@@ -535,10 +541,12 @@ def move_to(self, thing, destination):
                 t.location = destination
         return thing.bump
 
-    def add_thing(self, thing, location=(1, 1), exclude_duplicate_class_items=False):
+    def add_thing(self, thing, location=None, exclude_duplicate_class_items=False):
```

Comment:
Whenever this parameter is `None`, it is somehow causing the tests to fail while the default parameter `(1,1)` works fine with these modifications. I tried to debug the `defualt_location` method but it seems fine! I am a little confused why the tests are failing...

Reply:
Alright, due to the random placement of things in the Wumpus agent environment, the agent dies due to this line:
https://github.com/aimacode/aima-python/blob/fbdb36d8521e4ac8b1711e5c6e5f2c62955b8baa/agents.py#L958

I think the only choice is to let the parameter `location` as is for now.